services.yml

```diff
@@ -0,0 +1,19 @@
+version: '2'
+
+services:
+  base:
+    build: .
+    image: inspirehep/inspire-next
+  indexer:
+    image: elasticsearch
+    command: elasticsearch -Dcluster.name="inspire"
```

Comment:
Is there some preparatory steps where one can install icu and reindexing plugin?